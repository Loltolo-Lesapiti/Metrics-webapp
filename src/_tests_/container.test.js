import renderer from 'react-test-renderer';
import Forexcontainer from './mock/container';

describe('Currency display container', () => {
  it('Confirm that the currency display renders correctly', () => {
    const tree = renderer.create(<Forexcontainer />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
