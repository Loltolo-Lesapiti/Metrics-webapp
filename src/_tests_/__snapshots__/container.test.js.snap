// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Currency display container Confirm that the currency display renders correctly 1`] = `
Array [
  <h3>
    Currency Exchange
  </h3>,
  <div
    className="list-group"
    onKeyDown={[Function]}
  >
    <div
      className="list-group-item"
      data-rr-ui-event-key={null}
      onClick={[Function]}
    >
      <div
        className="card"
      >
        <div
          className="cardbady card-body"
        >
          <div
            className="card-title h5"
          />
          <p
            className="cardtext card-text"
          >
            The bidding price is
             
            . Click on the currency to view more details.
          </p>
        </div>
      </div>
    </div>
  </div>,
]
`;
